library/alloc/src/bstr.rs

```diff
@@ -188,6 +192,10 @@ impl Default for ByteString {
 //
 // #[unstable(feature = "bstr", issue = "134915")]
 // impl<const N: usize> From<[u8; N]> for ByteString {
+//     /// Make a `ByteString` from a byte array.
+//     ///
+//     /// ## Cost
+//     /// Allocates a new `Vec`
```

Comment:
This PR is inviting me to spend a lot of time on reviewing changes that it is not clear will ever actually become a functional difference in the repository's actual produced artifacts.